src/util-device.c

```diff
@@ -90,11 +90,12 @@ int LiveGetOffload(void)
  * to be created during runmode init, use LiveRegisterDevice()
  *
  *  \param dev string with the device name
+ *  \param copy_dev string with the name of the device being copied to (valid in IPS mode only)
  *
  *  \retval 0 on success.
  *  \retval -1 on failure.
  */
-int LiveRegisterDeviceName(const char *dev)
+int LiveRegisterDeviceName(const char *dev, const char *copy_dev)
```

Comment:
I'd suggest adding a new function ... e.g., `LiveRegisterDeviceNameWithCopy` and remove the changes to the source/runmode for the packet capture code. It's a bit complicated and one was missed (ipfw)